fabtests/common/shared.c

```diff
@@ -747,6 +747,12 @@ int ft_init(void)
 	tx_cq_cntr = 0;
 	rx_cq_cntr = 0;
 
+	int ret = ft_startup();
```

Comment:
Please declare variables at the top of the function and not inline with calls.  That makes them hard to find.